tklock.c

```diff
@@ -294,6 +304,82 @@ static void disable_autorelock_policy(void)
 	return;
 }
 
+/**
+ * Enable/disable X11 input events (currently only for touchscreen)
+ *
+ * @param enable TRUE enable events, FALSE disable events
+ * @return TRUE on success, FALSE on failure
+ */
+static gboolean generic_x11_event_control(const gboolean enable)
+{
+	gboolean ret = FALSE;
+	Display* dpy = NULL;
+	XDeviceInfo *devinfo = NULL;
+	int ndev = 0;
+	unsigned char data = (unsigned char)enable;
+
+	dpy = XOpenDisplay(NULL);
+	if (dpy == NULL) {
+		dpy = XOpenDisplay(":0.0");
+	}
```

Comment:
Curly braces here differ from the code style below for single command in an if statement.

Reply:
Will fix